libs/ui/inputs/src/lib/text-area/text-area.component.ts

```diff
@@ -24,6 +31,30 @@ export class TextAreaComponent implements AfterViewInit {
 
   @ViewChild('input') input
 
+  constructor() {
+    this.baseClasses = [
+      'w-full',
+      'pt-2',
+      'pl-2',
+      'resize-none',
+      'border',
+      'border-gray-800',
+      'rounded italic',
+      'leading-tight',
+      'focus:outline-none',
+      'focus:bg-background',
+      'focus:border-primary'
+    ].join(' ')
+
+    this.disabledClasses = [
+      'cursor-not-allowed'
+    ].join(' ')
+  }
+
+  get classList() {
+    return `${this.baseClasses} ${this.extraClass} ${this.disabled ? this.disabledClasses	: ''}`
+  }
+
```

Comment:
This can be done using a `disabled:` modified in TW: https://tailwindcss.com/docs/hover-focus-and-other-states#form-states